robot.py

```diff
@@ -1,23 +1,24 @@
 import asyncio
 from contextlib import asynccontextmanager
+import secrets
```

Comment:
This is a local import, rather than a built-in one. Move it to its own paragraph below the rest of the imports:
```python
...
from viam.robot.client import RobotClient

import secrets


@asynccontextmanager
async def create_robot...
```
For details, see the second bullet point of https://peps.python.org/pep-0008/#imports